src/docfx/Errors.cs

```diff
@@ -96,6 +96,9 @@ public static Error RedirectionDocumentIdConflict(IEnumerable<Document> redirect
         public static Error GitShadowClone(string repoPath)
             => new Error(ErrorLevel.Error, "git-shadow-clone", $"Does not support git shallow clone: '{repoPath}'");
 
+        public static Error GitNotFound()
+            => new Error(ErrorLevel.Error, "git-not-found", $"Cannot find git, install git https://git-scm.com/");
```

Comment:
>Error [](start = 36, length = 5)

Warning? Some users may not need git features.

Reply:
when this is executed, it sure is invoking git exe and the user is using git features that depend on git exe